src/System.Runtime.InteropServices.PInvoke/src/System.Runtime.InteropServices.PInvoke.builds

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Runtime.InteropServices.PInvoke.csproj">
```

Comment:
@weshaggard This is the same as it is in System.Runtime.InteropServices.builds. It is a full facade on net46 with the exception of the redirection calls for PInvokeMarshal, and a partial facade on netcore50aot and the empty target group.